frontend/components/account-page/AccountProfile.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+const AccountProfile = React.createClass({
+  getInitialState() {
+    return { account: this.props.account, edit: false };
+  },
+  componentWillReceiveProps(props) {
+    this.setState({ account: props.account });
+  },
+  handleChange(name, event) {
+    const value = event.target.value;
+
+    this.setState({
+      account: Object.assign({}, this.state.account, { [name]: value })
+    });
+  },
+  allowEdit() {
+    this.setState({ account: this.props.account, edit: true });
+  },
+  finishEdit(event) {
+    event.preventDefault();
+
+    const accountState = this.state.account;
+    const id = this.props.account.id;
+
+    this.props.editUserData(accountState, id);
+    this.setState({ edit: false });
+  },
+  render() {
+    if (this.state.edit === false) {
+      return (
+        <div>
+          <div className="account-user-info">
+            <h3>Username: {this.state.account.username}</h3>
+            <h3>Email: {this.state.account.email}</h3>
+            <h3>Level: {this.state.account.level}</h3>
+            <h3>Rating: {this.state.account.rating}</h3>
+            <h3>Biography: {this.state.account.bio}</h3>
+            <button className="account-edit-button"
+              onClick={this.allowEdit}>Edit</button>
```

Comment:
structure this differently. move the word 'Edit' to the next line and then move the closing tag for button under that.